Kitodo/src/main/java/de/sub/goobi/forms/ProzessverwaltungForm.java

```diff
@@ -1873,8 +1873,8 @@ public void createXML() {
     }
 
     /**
-     * transforms xml logfile with given xslt and provides download.
-     * //TODO: why this whole stuff is not used?
+     * transforms xml logfile with given xslt and provides download. //TODO: why
```

Comment:
This `TODO` should stay in front of a new line. Maybe you must add a empty line to reach this goal.

Reply:
Javadoc is HTML, so empty lines does not count here. The code formatter does it correcty. Question is, if this should be there at all. What did it mean?
> Why this whole stuff is not used?

Reply:
Research shows that this is part of a feature that is implemented incompletely. Currently, the XML download button downloads the XML log to the user’s work directory only. Here, the approach is to apply a custom XSLT selected in the front end, and download the results. There is no support for this in the front end. Keep method and fix front end or remove method?

Reply:
If this is not used in any previous version then we can remove it.